### ISSUE TITLE
Add Explainer prototype

### ISSUE THREAD

**Issue by sylvinus** [commonsearch/cosr-back#1]
The `explainer` is a web service for debugging query results.

It should be useful in these usecases:

- Enter an URL, fetch it, parse it like if it was just extracted from a WARC file and output a representation of the internal `HTMLDocument`, to view which parts of the page we ignore or consider as boilerplate.
- Enter a search + a URL/domain/docid and view the way the static rank was computed (from all the signals) and the Elasticsearch relevance debug info. This should be able completely explain the order of the search results.

I'm still cleaning an early prototype of this, and will push soon.

**Comment by mlinksva** [comment]
Great! I'm not sure right place to give feedback on explainer UI but two *tiny* things would be nice

- access to documentation for explainer output, maybe there's an elasticsearch help page that would cover this?
- 'About' link to project

**Comment by sylvinus** [comment]
Yes! Still definitely very rough. I just pushed an overview here: https://about.commonsearch.org/developer/result-quality

I've not yet found a good documentation of the ES/Lucene explain output online. Will reach out to ES devs about this.

Unrelated, but there are some good ideas there too:
https://github.com/polyfractal/elasticsearch-inquisitor

**Comment by sylvinus** [comment]
A few more features I'd like to have:
- a name="[docid]" for easy linking to documents in results
- a way to easily compare 2 explain outputs? It's hard to scroll down and up all the time.
